docs/community/community.md

```diff
@@ -1,6 +1,41 @@
 # KubeRay Community Governance
 
-## Becoming a Committer
+There are 4 roles in the KubeRay community:
+
+* **Contributor**: A contributor is a member who contributes to the KubeRay project through code, documentation,
+PR reviews, answering questions, etc.
+* **Triager**: A triager has the permission to triage issues and PRs.
+* **Approver**: An approver can approve PRs but cannot merge them.
```

Comment:
Since approvers cannot merge PRs, maybe `Reviewer` is a better title for this role

Reply:
Just saw K8s definitions: https://kubernetes.io/docs/contribute/participate/roles-and-responsibilities/#approvers.

Reply:
updated